vmm/src/cpu.rs

```diff
@@ -787,6 +832,26 @@ impl CpuManager {
         Ok(cpuid)
     }
 
+    #[cfg(target_arch = "aarch64")]
+    fn start_vcpu(
+        &mut self,
+        _cpu_id: u8,
+        _creation_ts: std::time::Instant,
+        _vcpu_thread_barrier: Arc<Barrier>,
+        _entry_point: Option<EntryPoint>,
+        _inserting: bool,
+        _snapshot: Option<Snapshot>,
+    ) -> Result<()> {
+        // temporary code to mute "unused variable" warnings
```

Comment:
Put this as `unimplemented!` and then use `#[cfg_attr(target_arch = "aarch64", allow(dead_code)]` on the fields instead,

Reply:
Good idea!